src/WebJobs.Extensions.DurableTask/EtwEventSource.cs

```diff
@@ -484,6 +484,36 @@ private EtwEventSource()
             this.WriteEvent(230, TaskHub, AppName, SlotName, Resource, Attempt, DelayMs, Details, ExtensionVersion);
         }
 
+        [Event(231, Level = EventLevel.Warning, Version = 2)]
```

Comment:
These `[Event(...)]` attributes should set `Version = 1` since they are new.

Reply:
Fixed.